stake-pool/program/src/processor.rs

```diff
@@ -493,6 +567,11 @@ impl Processor {
             &vote_account_address,
         )?;
 
+        if meta.lockup != stake_program::Lockup::default() {
+            msg!("Invalid lockup exists on validator stake account");
```

Comment:
```suggestion
            msg!("Validator stake account has a lockup");
```

Reply:
Hehe, this message was a bit of future-proofing for when we have lockup-enabled stake pools, but I'll use this message